ui/documents/semantic-ui/templates/documents/form/FormFieldsContainer.jsx

```diff
@@ -4,6 +4,8 @@ import {
   MultilingualTextInput,
   FormikStateLogger,
   EDTFSingleDatePicker,
+  sanitizeInput,
+  validTags
```

Comment:
![Screenshot 2024-06-17 at 10 13 44](https://github.com/Narodni-repozitar/nr-docs/assets/646743/3d0e7b4d-a0d1-4885-801a-b0fe17c2b19d)

Is it really exported in some PR?

Reply:
https://github.com/oarepo/oarepo-ui/pull/194